src/components/notifications/46elks.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div class="form-text">
+        <p>
+            You find the API-Username and API-password at <a href="https://46elks.com/account">https://46elks.com/account</a>
```

Comment:
Please move this Helptext under both inputs, simplify it to `Can be found on ..` and use component translation (`/<vue-i18n ...`) instead of leaving it untranslated.